pytorch_lightning/utilities/enums.py

```diff
@@ -240,7 +239,6 @@ class _StrategyType(LightningEnum):
     DP = "dp"
     DDP = "ddp"
     DDP2 = "ddp2"
-    DDP_CPU = "ddp_cpu"
     DDP_SPAWN = "ddp_spawn"
     TPU_SPAWN = "tpu_spawn"
```

Comment:
shouldn't we also remove this? in a follow-up?

Reply:
ddp_cpu could still be passed in throw strategy flag https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/trainer/connectors/accelerator_connector.py#L251-L255
This PR only remove support of passing in by accelerator flag

Reply:
I was asking about "tpu_spawn"

Reply:
yeah we could remove that. 
@four4fish soon we should probably also remove that error message specific to ddp_cpu you pointed out, as it will just be an invalid strategy name like any other.

Reply:
Let's do that in a follow-up. TPU_SPAWN is used in the `interactive_compatible` error messaging methods and I don't feel comfortable changing the behavior in this PR.